src/molecules/clock.js

```diff
@@ -0,0 +1,37 @@
+import React, {useState, useEffect} from 'react';
+import Display from '../atoms/display'
+import Panel from '../atoms/panel'
+
+const Clock = () => {
+    const [time, setTime] = useState(new Date());
+    const [isDateVisible, setIsDateVisible] = useState(true)
+   
+
+    useEffect(() => {
+        const timerId = setInterval(() => tick(), 1000);
+        return function cleanup() {
+            clearInterval(timerId);
+        }
+    }, [time]);
+
+    function tick() {
+        setTime(new Date());
+    }

```

Comment:
this function is not used anywhere else apart from useEffect above, I'd suggest to move tick() body to the arrow function above.
`setInterval(() => { ... } ), 1000);`